stric/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@bun-examples/stric",
+  "module": "index.ts",
+  "devDependencies": {
+    "bun-types": "^0.5.0",
+    "@types/react": "^18.0.28"
+  },
+  "type": "module",
+  "dependencies": {
+    "@stricjs/jsx": "^1.0.4",
```

Comment:
This is a reminder, but you may want to use `latest` instead a specific version, so when the new version released, you don't have to create a PR to edit the version.

Reply:
Added :)